tests/kafkatest/sanity_checks/test_kafka_version.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.tests.test import Test
+
+from kafkatest.services.zookeeper import ZookeeperService
+from kafkatest.services.kafka import KafkaService
+from kafkatest.services.kafka import KafkaVersion
+
+
+class KafkaVersionTest(Test):
```

Comment:
Why are these only sanity checks? Because we don't actually send any data through them?

Right now I'm not clear on the split between sanity checks and normal tests. For example, `test_console_consumer.py` seems like a pretty complete test with consumer pinned to 0.8.2.1 and everything else on trunk; it produces, consumes, and validates the messages.